checker-framework-eclipse-update-site/manual/index.html

```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Can you remove the binary `.jar` files from the commit?
Aren't these files rebuilt automatically?

Reply:
Removed. Should I also remove the log.zip file?

Reply:
Sure, committing a log doesn't seem useful.